py/vttest/local_database.py

```diff
@@ -27,7 +27,12 @@ def setup(self):
     if self.mysql_only:
       return
 
-    vt_processes.start_vt_processes(self.directory, self.shards, self.mysql_db)
+    vschema = ''
+    if self.schema_dir:
+      path = os.path.join(self.schema_dir, "vschema.json")
+      if os.path.isfile(path):
+        vschema = path
+    vt_processes.start_vt_processes(self.directory, self.shards, self.mysql_db, vschema)
```

Comment:
let's make vschema a command line parameter, instead of hardcoding the file name in the schema dir?